mapper/orm-coordination-outbox-polling/src/main/java/org/hibernate/search/mapper/orm/coordination/outboxpolling/cluster/impl/OutboxPollingAgentAdditionalJaxbMappingProducer.java

```diff
@@ -72,23 +74,17 @@ public class OutboxPollingAgentAdditionalJaxbMappingProducer
 			"        </property>\n" +
 			"        <property name=\"totalShardCount\" nullable=\"true\" />\n" +
 			"        <property name=\"assignedShardIndex\" nullable=\"true\" />\n" +
-			// Reserved for future use
-			"        <property name=\"payload\" nullable=\"true\" type=\"materialized_blob\">\n" +
-			// HSEARCH-4727: this column length will be ignored in most dialects, since the blob type is normally unbounded,
-			// but it will force Hibernate ORM to simulate an unbounded BLOB type with DB2.
-			// Using 2147483647 as it's the documented maximum length of BLOBs in DB2:
-			// https://www.ibm.com/docs/en/db2-for-zos/11?topic=types-large-objects-lobs
-			// TODO HSEARCH-4395/HSEARCH-4532 drop this length definition with ORM 6, because ORM 6 will ignore it.
-			"                <column length=\"2147483647\" />\n" +
-			"        </property>\n" +
+			// Payload column (reserved for future use):
+			"        %6$s\n" +
 			"    </class>\n" +
 			"</hibernate-mapping>\n";
 
 	public static final String ENTITY_DEFINITION = String.format(
 			Locale.ROOT, ENTITY_DEFINITION_TEMPLATE, "", "",
 			HibernateOrmMapperOutboxPollingSettings.Defaults.COORDINATION_ENTITY_MAPPING_AGENT_TABLE,
 			HibernateOrmMapperOutboxPollingSettings.Defaults.COORDINATION_ENTITY_MAPPING_AGENT_UUID_GEN_STRATEGY,
-			UuidDataTypeUtils.UUID_CHAR
+			UuidDataTypeUtils.UUID_CHAR,
+			PayloadMappingUtils.payload( PayloadType.MATERIALIZED_BLOB, true )
```

Comment:
I think this should be `PayloadType.LONG32VARBINARY`?

Reply:
😃 I wasn't sure which one we wanted here, but yeah, probably a "default" one would be the correct one to go with. Thanks! Made it pointing to the `HibernateOrmMapperOutboxPollingSettings.Defaults`